src/components/Screenshot/index.ts

```diff
@@ -0,0 +1,10 @@
+export { default as ScreenshotReady } from './ScreenshotReady';
+
+/**
+ * This class name must be used to mark content that should be included in
+ * screenshotting purposes.
+ *
+ * It should be paired with a <ScreenshotReady /> element, which indicates when
+ * the content is fully-rendered and ready for the screenshot to be taken.
+ */
+export const SCREENSHOT_CLASS = 'screenshot';
```

Comment:
Maybe we can simplify this a bit by creating a component that renders both the wrapper and the sentinel at once:

```tsx
const ScreenshotWrapper: React.FC = ({ children }) => (
  <div className='screenshot'>
    {children}
    <div className='screenshot-ready' style={{display: 'none'}} />
  </div>
);

// later...

<ScreenshotWrapper>
  <SomeChart {...props} />
</ScreenshotWrapper>
```

Reply:
Yeah...  I originally tried to make this cleaner but I ran into two problems:

1. If we have a ScreenshotWrapper component and then you try to style it with styled-components (which _seems_ like a totally normal / sane thing to do, then the className gets lost (it gets minified or omitted or something) and so things break.
2. The reason we need the screenshot-ready div is because we can't rely on all of the children populating synchronously.  In particular, we don't know when the projections are done loading and the chart has rendered at its final width.  So it's a bit hacky, but right now you need to carefully place the `<ScreenshotReady>` component somewhere that's deeply nested enough that it'll only render once everything is actually ready.  So e.g. for the charts, we place it here: https://github.com/covid-projections/covid-projections/pull/1370/files/5882001a1fa82d49db87f3b3189741a38a904611#diff-1f39dac4db94a4170cf8add355312ee6R73

Reply:
oh I see, that makes sense